src/components/shared/SpentTime/SpentTime.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import { useFirestore } from "reactfire";
+import { collection, getDocs, query, where } from "firebase/firestore";
+import { chunk } from "lodash";
+
+import {
+  COLLECTION_SPACES,
+  FIRESTORE_QUERY_IN_ARRAY_MAX_ITEMS,
+} from "settings";
+
+import { getUserRef } from "api/profile";
+
+import { UserVisit } from "types/Firestore";
+import { UserId } from "types/id";
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+import { isTruthy } from "utils/types";
+
+import { useWorldAndSpaceByParams } from "hooks/spaces/useWorldAndSpaceByParams";
+
+import styles from "./SpentTime.module.scss";
```

Comment:
[NOTE] 

These aren't styles, these are classes, a name mapping from human readable to Webpack generated.

In other words it is an object of `Record<string, string>` type which is different from `CSSStyleDeclaration` one (https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration)

Reply:
renamed the constants here: https://github.com/sparkletown/sparkle/pull/2953/commits/9567a98c837f8f4ff16dfba4f38bb70ea2929ab8